### PR TITLE
Drop `fit-textarea` dependency

### PR DESCRIPTION
**Pull request by fregante**
## What does this PR do?

Small PR, removes my own dependency for performance

- Closes https://github.com/pixiebrix/pixiebrix-extension/issues/8972

## Notes

- This bumps the minimum Chrome version to v123 (the latest is v130 now)
- Change untested

Feel free to close or postpone if it's not time yet.

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -5,11 +5,11 @@
   "short_name": "PixieBrix",
   "description": "Customize, extend, and integrate your favorite SaaS apps without writing a line of code!",
   "homepage_url": "https://www.pixiebrix.com",
-  "minimum_chrome_version": "116.0",
+  "minimum_chrome_version": "123.0",
```
Comment (fregante) [inline src/manifest.json:8]: This also enables:

https://github.com/pixiebrix/pixiebrix-extension/blob/cda09cb0c240f50ec676f41612af184f168c0da2/src/utils/urlUtils.ts#L111

**Review by fregante — COMMENTED** [review]
There's a number of snapshot tests that need to be updated too. I can do it if the PR is accepted

File: src/manifest.json [hunk 2]
```diff
@@ -5,11 +5,11 @@
...
   "browser_specific_settings": {
     "gecko": {
       "id": "extension@pixiebrix.com",
-      "strict_min_version": "91.0"
+      "strict_min_version": "123.0"
```
Comment (grahamlangford) [inline src/manifest.json:12]: We no longer support Firefox, should we just remove this property? Also, is the minimum version really the same between firefox and chrome?

File: src/manifest.json (hunk 1 above)
Comment (grahamlangford) [reply src/manifest.json:8]: I'm supportive in principle. We officially only support the last two major versions of Chrome, but I'll need to confirm with @twschiller and @Pashaminkovsky before bumping the minimum chrome version

Comment (grahamlangford) [reply src/manifest.json:8]: NOTE: I checked Mixpanel and it appears that the vast majority of our users are on Chrome 128 or Chrome 129

File: src/manifest.json (hunk 2 above)
Comment (fregante) [reply src/manifest.json:12]: Indeed I forgot to remove this key in #7237 and should probably be dropped.

File: src/manifest.json (hunk 1 above)
Comment (grahamlangford) [reply src/manifest.json:8]: Per @Pashaminkovsky, we're comfortable with the version bump. We do need to properly test this PR

**Comment by fregante** [comment]
Closing because of how it interacts with flexbox areas. They would have to be reviewed so that each `textarea` doesn't try to grow horizontally. Particularly, the pieces on each side of the `textarea` would need to have `flex-shrink: 0` probably:

varpopup

This requires extra exploration/work. Closing

**Comment by fregante** [comment]
By the way I'm see this issue:

varpopup

**Comment by grahamlangford** [comment]
Thanks @fregante. That FAB issue is very concerning. We'll take a look immediately

**Pull request closed without merging**